collect_app/src/main/java/org/odk/collect/android/fragments/FormManagerList.java

```diff
@@ -160,6 +163,12 @@ public void onClick(DialogInterface dialog, int i) {
     private void deleteSelectedForms() {
         // only start if no other task is running
         if (backgroundTasks.deleteFormsTask == null) {
+
+            Bundle args = new Bundle();
+            args.putSerializable(ProgressDialogFragment.MESSAGE, getResources().getString(R.string.form_delete_message));
+            args.putSerializable(ProgressDialogFragment.CANCELABLE, "true");
```

Comment:
You can just do `putBoolean` here which will be easier on the other side!

Reply:
Also could you add a test for this new functionality to `ProgressDialogFragment`?

Reply:
Yes that would be great! I'll add tests. Also, I wanted to ask, should we use `ProgressDialogFragment` in `DeleteSavedForms Fragment`, as well?

Reply:
@SaumiaSinghal probably a good idea down the line but no need to do it as part of this PR